CHANGELOG.md

```diff
@@ -21,8 +21,15 @@ The following improvements might require changes to how you use rustup:
 
 rustup now officially supports the following host platforms:
 
-  - `aarch64-pc-windows-msvc` [pr#3840]
-  - `loongarch64-unknown-linux-musl` [pr#3921]
+- `aarch64-pc-windows-msvc` [pr#3840]
+- `loongarch64-unknown-linux-musl` [pr#3921]
+
+This release also comes with various quality-of-life improvements, to name a few:
+
+- `rustup show`'s output format has been refurbished, making it easier to find out about your toolchains' status. [pr#3225]
```

Comment:
Nit: "refurbished" seems like an odd word choice to me for this, maybe "cleaned up", or "reformatted"?

Reply:
@djc Done!